src/igvfd/schemas/changelogs/biosample.md

```diff
@@ -1,5 +1,9 @@
 ## Changelog for *`biosample.json`*
 
+### Schema version 2
+
+* Added `donors` and `taxa` to requirements.
```

Comment:
We have decided to use present tense in the changelogs, so we should change "added" to "add" for consistency. This will affect all the other changelogs in this PR as well.